tracee-rules/signature.go

```diff
@@ -12,11 +14,10 @@ import (
 	"plugin"
 	"strings"
 
-	"github.com/aquasecurity/tracee/tracee-rules/signatures/rego/regosig"
 	"github.com/aquasecurity/tracee/tracee-rules/types"
 )
 
-//go:embed signatures/rego/helpers.rego
+//go:embed helpers.rego
```

Comment:
This duplication is needed as `go embed` cannot work with relative paths (and symlinks).

Reply:
To get rid of duplicates in `signatures/rego/helpers.rego` and `tracee-rules/helpers.rego` you can created the `embedded.go` file in the root of the Git repo / Go module with the following content:

```
package tracee

import _ "embed"

//go:embed signatures/rego/helpers.rego
var RegoHelpersCode string
```

Then you. can use import and use the `RegoHelpersCode` variable in `signature.go`. I'm using this approach in https://github.com/aquasecurity/starboard/blob/main/embedded.go because of the limitations you mentioned.